# Titanic competition with TensorFlow Decision Forests

This notebook will take you through the steps needed to train a baseline Gradient Boosted Trees Model using TensorFlow Decision Forests and creating a submission on the Titanic competition. 

This notebook shows:

1. How to do some basic pre-processing. For example, the passenger names will be tokenized, and ticket names will be splitted in parts.
1. How to train a Gradient Boosted Trees (GBT) with default parameters
1. How to train a GBT with improved default parameters
1. How to tune the parameters of a GBTs
1. How to train and ensemble many GBTs

# Imports dependencies

In [14]:
import numpy as np
import pandas as pd
import os

import tensorflow as tf
import tensorflow_decision_forests as tfdf
#Disable warning
import warnings
warnings.filterwarnings("ignore")

print(f"Found TF-DF {tfdf.__version__}")

Found TF-DF 1.5.0


# Load dataset

In [15]:
train_df = pd.read_csv("Dataset/train.csv")
serving_df = pd.read_csv("Dataset/test.csv")

train_df.head(10)

PassengerId  Survived  Pclass  \
0            1         0       3   
1            2         1       1   
2            3         1       3   
3            4         1       1   
4            5         0       3   
5            6         0       3   
6            7         0       1   
7            8         0       3   
8            9         1       3   
9           10         1       2   

                                                Name     Sex   Age  SibSp  \
0                            Braund, Mr. Owen Harris    male  22.0      1   
1  Cumings, Mrs. John Bradley (Florence Briggs Th...  female  38.0      1   
2                             Heikkinen, Miss. Laina  female  26.0      0   
3       Futrelle, Mrs. Jacques Heath (Lily May Peel)  female  35.0      1   
4                           Allen, Mr. William Henry    male  35.0      0   
5                                   Moran, Mr. James    male   NaN      0   
6                            McCarthy, Mr. Timothy J    male  54.0      0   
7                     Palsson, Master. Gosta Leonard    male   2.0      3   
8  Johnson, Mrs. Oscar W (Elisabeth Vilhelmina Berg)  female  27.0      0   
9                Nasser, Mrs. Nicholas (Adele Achem)  female  14.0      1   

   Parch            Ticket     Fare Cabin Embarked  
0      0         A/5 21171   7.2500   NaN        S  
1      0          PC 17599  71.2833   C85        C  
2      0  STON/O2. 3101282   7.9250   NaN        S  
3      0            113803  53.1000  C123        S  
4      0            373450   8.0500   NaN        S  
5      0            330877   8.4583   NaN        Q  
6      0             17463  51.8625   E46        S  
7      1            349909  21.0750   NaN        S  
8      2            347742  11.1333   NaN        S  
9      0            237736  30.0708   NaN        C

# Prepare dataset

We will apply the following transformations on the dataset.

1. Tokenize the names. For example, "Braund, Mr. Owen Harris" will become ["Braund", "Mr.", "Owen", "Harris"].
2. Extract any prefix in the ticket. For example ticket "STON/O2. 3101282" will become "STON/O2." and 3101282.

In [16]:
def preprocess(df):
    df = df.copy()
    
    def normalize_name(x):
        return " ".join([v.strip(",()[].\"'") for v in x.split(" ")])
    
    def ticket_number(x):
        return x.split(" ")[-1]
        
    def ticket_item(x):
        items = x.split(" ")
        if len(items) == 1:
            return "NONE"
        return "_".join(items[0:-1])
    
    df["Name"] = df["Name"].apply(normalize_name)
    df["Ticket_number"] = df["Ticket"].apply(ticket_number)
    df["Ticket_item"] = df["Ticket"].apply(ticket_item)                     
    return df
    
preprocessed_train_df = preprocess(train_df)
preprocessed_serving_df = preprocess(serving_df)

preprocessed_train_df.head(5)

PassengerId  Survived  Pclass  \
0            1         0       3   
1            2         1       1   
2            3         1       3   
3            4         1       1   
4            5         0       3   

                                              Name     Sex   Age  SibSp  \
0                            Braund Mr Owen Harris    male  22.0      1   
1  Cumings Mrs John Bradley Florence Briggs Thayer  female  38.0      1   
2                             Heikkinen Miss Laina  female  26.0      0   
3         Futrelle Mrs Jacques Heath Lily May Peel  female  35.0      1   
4                           Allen Mr William Henry    male  35.0      0   

   Parch            Ticket     Fare Cabin Embarked Ticket_number Ticket_item  
0      0         A/5 21171   7.2500   NaN        S         21171         A/5  
1      0          PC 17599  71.2833   C85        C         17599          PC  
2      0  STON/O2. 3101282   7.9250   NaN        S       3101282    STON/O2.  
3      0            113803  53.1000  C123        S        113803        NONE  
4      0            373450   8.0500   NaN        S        373450        NONE

Let's keep the list of the input features of the model. Notably, we don't want to train our model on the "PassengerId" and "Ticket" features.

In [17]:
input_features = list(preprocessed_train_df.columns)
input_features.remove("Ticket")
input_features.remove("PassengerId")
input_features.remove("Survived")
#input_features.remove("Ticket_number")

print(f"Input features: {input_features}")

Input features: ['Pclass', 'Name', 'Sex', 'Age', 'SibSp', 'Parch', 'Fare', 'Cabin', 'Embarked', 'Ticket_number', 'Ticket_item']


# Convert Pandas dataset to TensorFlow Dataset

In [18]:
def tokenize_names(features, labels=None):
    """Divite the names into tokens. TF-DF can consume text tokens natively."""
    features["Name"] =  tf.strings.split(features["Name"])
    return features, labels

train_ds = tfdf.keras.pd_dataframe_to_tf_dataset(preprocessed_train_df,label="Survived").map(tokenize_names)
serving_ds = tfdf.keras.pd_dataframe_to_tf_dataset(preprocessed_serving_df).map(tokenize_names)

# Train model with default parameters

### Train model

First, we are training a GradientBoostedTreesModel model with the default parameters.

In [19]:
model = tfdf.keras.GradientBoostedTreesModel(
    verbose=0, # Very few logs
    features=[tfdf.keras.FeatureUsage(name=n) for n in input_features],
    exclude_non_specified_features=True, # Only use the features in "features"
    random_seed=1234,
)
model.fit(train_ds)

self_evaluation = model.make_inspector().evaluation()
print(f"Accuracy: {self_evaluation.accuracy} Loss:{self_evaluation.loss}")

[WARNING 23-11-08 14:10:54.5686 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:10:54.5744 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:10:54.5744 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


[INFO 23-11-08 14:10:56.9882 NZDT kernel.cc:1243] Loading model from path /tmp/tmpikpb21ex/model/ with prefix 80394b53d1804cb9
[INFO 23-11-08 14:10:57.0103 NZDT abstract_model.cc:1311] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 23-11-08 14:10:57.0103 NZDT kernel.cc:1075] Use fast generic engine


Accuracy: 0.8260869383811951 Loss:0.8608942627906799


# Train model with improved default parameters

Now you'll use some specific parameters when creating the GBT model

In [20]:
model = tfdf.keras.GradientBoostedTreesModel(
    verbose=0, # Very few logs
    features=[tfdf.keras.FeatureUsage(name=n) for n in input_features],
    exclude_non_specified_features=True, # Only use the features in "features"
    
    #num_trees=2000,
    
    # Only for GBT.
    # A bit slower, but great to understand the model.
    # compute_permutation_variable_importance=True,
    
    # Change the default hyper-parameters
    # hyperparameter_template="benchmark_rank1@v1",
    
    #num_trees=1000,
    #tuner=tuner
    
    min_examples=1,
    categorical_algorithm="RANDOM",
    #max_depth=4,
    shrinkage=0.05,
    #num_candidate_attributes_ratio=0.2,
    split_axis="SPARSE_OBLIQUE",
    sparse_oblique_normalization="MIN_MAX",
    sparse_oblique_num_projections_exponent=2.0,
    num_trees=2000,
    #validation_ratio=0.0,
    random_seed=1234,
    
)
model.fit(train_ds)

self_evaluation = model.make_inspector().evaluation()
print(f"Accuracy: {self_evaluation.accuracy} Loss:{self_evaluation.loss}")

[WARNING 23-11-08 14:10:57.6830 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:10:57.6831 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:10:57.6831 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


[INFO 23-11-08 14:10:59.1115 NZDT kernel.cc:1243] Loading model from path /tmp/tmpwx8vx9yz/model/ with prefix feecb0f7649c44be
[INFO 23-11-08 14:10:59.1171 NZDT decision_forest.cc:660] Model loaded with 42 root(s), 2212 node(s), and 10 input feature(s).
[INFO 23-11-08 14:10:59.1171 NZDT kernel.cc:1075] Use fast generic engine


Accuracy: 0.782608687877655 Loss:1.060815453529358


Let's look at the model and you can also notice the information about variable importance that the model figured out

In [21]:
model.summary()

Model: "gradient_boosted_trees_model_5"
_________________________________________________________________
 Layer (type)                Output Shape              Param #   


Total params: 1 (1.00 Byte)
Trainable params: 0 (0.00 Byte)
Non-trainable params: 1 (1.00 Byte)
_________________________________________________________________
Type: "GRADIENT_BOOSTED_TREES"
Task: CLASSIFICATION
Label: "__LABEL"

Input Features (11):
	Age
	Cabin
	Embarked
	Fare
	Name
	Parch
	Pclass
	Sex
	SibSp
	Ticket_item
	Ticket_number

No weights

Variable Importance: INV_MEAN_MIN_DEPTH:
    1.           "Sex"  0.597073 ################
    2.           "Age"  0.363764 #######
    3.          "Fare"  0.264018 ###
    4.          "Name"  0.207843 #
    5.        "Pclass"  0.178906 
    6. "Ticket_number"  0.178488 
    7.   "Ticket_item"  0.177907 
    8.      "Embarked"  0.177237 
    9.         "Parch"  0.175481 
   10.         "SibSp"  0.171800 

Variable Importance: NUM_AS_ROOT:
    1.  "Sex" 36.000000 ################
    2. "Name"  6.000000 

Variable Importance: NUM_NODES:
    1.           "Age" 530.000000 ################
    2.          "Fare" 311.000000 #########
    3.  

# Make predictions

In [22]:
def prediction_to_kaggle_format(model, threshold=0.5):
    proba_survive = model.predict(serving_ds, verbose=0)[:,0]
    return pd.DataFrame({
        "PassengerId": serving_df["PassengerId"],
        "Survived": (proba_survive >= threshold).astype(int)
    })

def make_submission(kaggle_predictions):
    path="Dataset/submission.csv"
    kaggle_predictions.to_csv(path, index=False)
    print(f"Submission exported to {path}")
    
kaggle_predictions = prediction_to_kaggle_format(model)
make_submission(kaggle_predictions)
!head Dataset/submission.csv

Submission exported to Dataset/submission.csv
PassengerId,Survived
892,0
893,0
894,0
895,0
896,0
897,0
898,0
899,0
900,1


# Training a model with hyperparameter tunning

Hyper-parameter tuning is enabled by specifying the tuner constructor argument of the model. The tuner object contains all the configuration of the tuner (search space, optimizer, trial and objective).


In [23]:
tuner = tfdf.tuner.RandomSearch(num_trials=1000)
tuner.choice("min_examples", [2, 5, 7, 10])
tuner.choice("categorical_algorithm", ["CART", "RANDOM"])

local_search_space = tuner.choice("growing_strategy", ["LOCAL"])
local_search_space.choice("max_depth", [3, 4, 5, 6, 8])

global_search_space = tuner.choice("growing_strategy", ["BEST_FIRST_GLOBAL"], merge=True)
global_search_space.choice("max_num_nodes", [16, 32, 64, 128, 256])

#tuner.choice("use_hessian_gain", [True, False])
tuner.choice("shrinkage", [0.02, 0.05, 0.10, 0.15])
tuner.choice("num_candidate_attributes_ratio", [0.2, 0.5, 0.9, 1.0])


tuner.choice("split_axis", ["AXIS_ALIGNED"])
oblique_space = tuner.choice("split_axis", ["SPARSE_OBLIQUE"], merge=True)
oblique_space.choice("sparse_oblique_normalization",
                     ["NONE", "STANDARD_DEVIATION", "MIN_MAX"])
oblique_space.choice("sparse_oblique_weights", ["BINARY", "CONTINUOUS"])
oblique_space.choice("sparse_oblique_num_projections_exponent", [1.0, 1.5])

# Tune the model. Notice the `tuner=tuner`.
tuned_model = tfdf.keras.GradientBoostedTreesModel(tuner=tuner)
tuned_model.fit(train_ds, verbose=0)

tuned_self_evaluation = tuned_model.make_inspector().evaluation()
print(f"Accuracy: {tuned_self_evaluation.accuracy} Loss:{tuned_self_evaluation.loss}")

Use /tmp/tmpzmlmaa55 as temporary training directory


[WARNING 23-11-08 14:11:00.8369 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:11:00.8370 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:11:00.8370 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:12:20.6784 NZDT kernel.cc:1243] Loading model from path /tmp/tmpzmlmaa55/model/ with prefix 1290528c902547e2
[INFO 23-11-08 14:12:20.6927 NZDT decision_forest.cc:660] Model loaded with 32 root(s), 852 node(s), and 12 input feature(s).
[INFO 23-11-08 14:12:20.6927 NZDT abstract_model.cc:1311] Engine "GradientBoostedTreesGeneric" built
[INFO 23-11-08 14:12:20.6927 NZDT kernel.cc:1075] Use fast generic engine


Accuracy: 0.8630136847496033 Loss:0.6804219484329224


In the last line in the cell above, you can see the accuracy is higher than previously with default parameters and parameters set by hand.

This is the main idea behing hyperparameter tuning.

For more information you can follow this tutorial: [Automated hyper-parameter tuning](https://www.tensorflow.org/decision_forests/tutorials/automatic_tuning_colab)

# Making an ensemble

Here you'll create 100 models with different seeds and combine their results

This approach removes a little bit the random aspects related to creating ML models

In the GBT creation is used the `honest` parameter. It will use different training examples to infer the structure and the leaf values. This regularization technique trades examples for bias estimates.

In [24]:
predictions = None
num_predictions = 0

for i in range(100):
    print(f"i:{i}")
    # Possible models: GradientBoostedTreesModel or RandomForestModel
    model = tfdf.keras.GradientBoostedTreesModel(
        verbose=0, # Very few logs
        features=[tfdf.keras.FeatureUsage(name=n) for n in input_features],
        exclude_non_specified_features=True, # Only use the features in "features"

        #min_examples=1,
        #categorical_algorithm="RANDOM",
        ##max_depth=4,
        #shrinkage=0.05,
        ##num_candidate_attributes_ratio=0.2,
        #split_axis="SPARSE_OBLIQUE",
        #sparse_oblique_normalization="MIN_MAX",
        #sparse_oblique_num_projections_exponent=2.0,
        #num_trees=2000,
        ##validation_ratio=0.0,
        random_seed=i,
        honest=True,
    )
    model.fit(train_ds)
    
    sub_predictions = model.predict(serving_ds, verbose=0)[:,0]
    if predictions is None:
        predictions = sub_predictions
    else:
        predictions += sub_predictions
    num_predictions += 1

predictions/=num_predictions

kaggle_predictions = pd.DataFrame({
        "PassengerId": serving_df["PassengerId"],
        "Survived": (predictions >= 0.5).astype(int)
    })

make_submission(kaggle_predictions)

i:0


[WARNING 23-11-08 14:12:21.0928 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:12:21.0929 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:12:21.0929 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:12:22.0291 NZDT kernel.cc:1243] Loading model from path /tmp/tmpi5b9hiec/model/ with prefix c7b92a0a699f4c78
[INFO 23-11-08 14:12:22.0327 NZDT kernel.cc:1075] Use fast generic engine


i:1


[WARNING 23-11-08 14:12:22.4133 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:12:22.4133 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:12:22.4133 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:12:23.7726 NZDT kernel.cc:1243] Loading model from path /tmp/tmpok7tqcvg/model/ with prefix 4a4b399a93c64e8a
[INFO 23-11-08 14:12:23.7845 NZDT kernel.cc:1075] Use fast generic engine


i:2


[WARNING 23-11-08 14:12:24.1579 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:12:24.1579 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:12:24.1579 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:12:24.8044 NZDT kernel.cc:1243] Loading model from path /tmp/tmp03mmu3si/model/ with prefix f695533a5ad54b8b
[INFO 23-11-08 14:12:24.8079 NZDT kernel.cc:1075] Use fast generic engine


i:3


[WARNING 23-11-08 14:12:25.2941 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:12:25.2941 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:12:25.2941 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:12:27.0632 NZDT kernel.cc:1243] Loading model from path /tmp/tmpbkt4z2z1/model/ with prefix 6b4ffb66e0e54d1b
[INFO 23-11-08 14:12:27.0960 NZDT kernel.cc:1075] Use fast generic engine


i:4


[WARNING 23-11-08 14:12:27.5008 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:12:27.5009 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:12:27.5009 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:12:28.2374 NZDT kernel.cc:1243] Loading model from path /tmp/tmp6wpa8kcq/model/ with prefix cbb286bfd0c54134
[INFO 23-11-08 14:12:28.2405 NZDT kernel.cc:1075] Use fast generic engine


i:5


[WARNING 23-11-08 14:12:28.5846 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:12:28.5846 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:12:28.5846 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:12:29.1953 NZDT kernel.cc:1243] Loading model from path /tmp/tmpnmqah54u/model/ with prefix 215d3081011d43c3
[INFO 23-11-08 14:12:29.1983 NZDT kernel.cc:1075] Use fast generic engine


i:6


[WARNING 23-11-08 14:12:29.6387 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:12:29.6387 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:12:29.6388 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:12:30.4464 NZDT kernel.cc:1243] Loading model from path /tmp/tmpyhc2ium7/model/ with prefix c7177ceeb70c41a8
[INFO 23-11-08 14:12:30.4511 NZDT kernel.cc:1075] Use fast generic engine


i:7


[WARNING 23-11-08 14:12:30.8192 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:12:30.8192 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:12:30.8192 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:12:32.3048 NZDT kernel.cc:1243] Loading model from path /tmp/tmp6dztqlgc/model/ with prefix 2b4e2a129425467a
[INFO 23-11-08 14:12:32.3476 NZDT abstract_model.cc:1311] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 23-11-08 14:12:32.3476 NZDT kernel.cc:1075] Use fast generic engine


i:8


[WARNING 23-11-08 14:12:32.7851 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:12:32.7851 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:12:32.7851 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:12:33.5927 NZDT kernel.cc:1243] Loading model from path /tmp/tmpmcj4l4p2/model/ with prefix 156c79fed05244be
[INFO 23-11-08 14:12:33.5982 NZDT kernel.cc:1075] Use fast generic engine


i:9


[WARNING 23-11-08 14:12:33.9414 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:12:33.9414 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:12:33.9414 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:12:35.0440 NZDT kernel.cc:1243] Loading model from path /tmp/tmpjiuo19v2/model/ with prefix 8e547754d2c14a18
[INFO 23-11-08 14:12:35.0534 NZDT kernel.cc:1075] Use fast generic engine


i:10


[WARNING 23-11-08 14:12:37.9156 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:12:37.9156 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:12:37.9156 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:12:38.5999 NZDT kernel.cc:1243] Loading model from path /tmp/tmpdua9wrqb/model/ with prefix effbdc8732c44595
[INFO 23-11-08 14:12:38.6040 NZDT kernel.cc:1075] Use fast generic engine


i:11


[WARNING 23-11-08 14:12:39.0120 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:12:39.0121 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:12:39.0121 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:12:40.0927 NZDT kernel.cc:1243] Loading model from path /tmp/tmp6aei2u4u/model/ with prefix 62c0d11637964564
[INFO 23-11-08 14:12:40.1009 NZDT kernel.cc:1075] Use fast generic engine


i:12


[WARNING 23-11-08 14:12:40.5176 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:12:40.5177 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:12:40.5177 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:12:41.1870 NZDT kernel.cc:1243] Loading model from path /tmp/tmp84x4qmb8/model/ with prefix 2787b214b19d42dd
[INFO 23-11-08 14:12:41.1906 NZDT kernel.cc:1075] Use fast generic engine


i:13


[WARNING 23-11-08 14:12:41.5460 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:12:41.5460 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:12:41.5460 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:12:42.4614 NZDT kernel.cc:1243] Loading model from path /tmp/tmp_rwzx1k5/model/ with prefix 938d7f078bd246b7
[INFO 23-11-08 14:12:42.4689 NZDT abstract_model.cc:1311] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 23-11-08 14:12:42.4690 NZDT kernel.cc:1075] Use fast generic engine


i:14


[WARNING 23-11-08 14:12:42.8888 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:12:42.8888 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:12:42.8888 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:12:43.6394 NZDT kernel.cc:1243] Loading model from path /tmp/tmp3j1e_is3/model/ with prefix 87273ba530c84aec
[INFO 23-11-08 14:12:43.6439 NZDT kernel.cc:1075] Use fast generic engine


i:15


[WARNING 23-11-08 14:12:44.0709 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:12:44.0709 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:12:44.0709 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:12:44.8855 NZDT kernel.cc:1243] Loading model from path /tmp/tmpahh4wb69/model/ with prefix e1d0847ff1a9444e
[INFO 23-11-08 14:12:44.8911 NZDT kernel.cc:1075] Use fast generic engine


i:16


[WARNING 23-11-08 14:12:45.2316 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:12:45.2317 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:12:45.2317 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:12:46.3061 NZDT kernel.cc:1243] Loading model from path /tmp/tmpm11l4f7x/model/ with prefix 2341818269ed4108
[INFO 23-11-08 14:12:46.3133 NZDT kernel.cc:1075] Use fast generic engine


i:17


[WARNING 23-11-08 14:12:46.7304 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:12:46.7304 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:12:46.7304 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:12:47.7631 NZDT kernel.cc:1243] Loading model from path /tmp/tmpjkm9aywz/model/ with prefix 8a9747ecee224d5d
[INFO 23-11-08 14:12:47.7715 NZDT kernel.cc:1075] Use fast generic engine


i:18


[WARNING 23-11-08 14:12:48.1516 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:12:48.1516 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:12:48.1517 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:12:49.1081 NZDT kernel.cc:1243] Loading model from path /tmp/tmppgzlfg7k/model/ with prefix 943f07734d624cf3
[INFO 23-11-08 14:12:49.1145 NZDT kernel.cc:1075] Use fast generic engine


i:19


[WARNING 23-11-08 14:12:49.5082 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:12:49.5082 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:12:49.5082 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:12:50.7836 NZDT kernel.cc:1243] Loading model from path /tmp/tmp6rzatsgm/model/ with prefix 858b54e997aa4e56
[INFO 23-11-08 14:12:50.7942 NZDT kernel.cc:1075] Use fast generic engine


i:20


[WARNING 23-11-08 14:12:51.2460 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:12:51.2461 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:12:51.2461 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:12:52.2777 NZDT kernel.cc:1243] Loading model from path /tmp/tmp6fqboq2n/model/ with prefix 8f43f21cbd5547d8
[INFO 23-11-08 14:12:52.2864 NZDT kernel.cc:1075] Use fast generic engine


i:21


[WARNING 23-11-08 14:12:52.7060 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:12:52.7060 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:12:52.7061 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:12:53.4018 NZDT kernel.cc:1243] Loading model from path /tmp/tmp42u3yuqe/model/ with prefix bac41a128ff0462d
[INFO 23-11-08 14:12:53.4050 NZDT abstract_model.cc:1311] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 23-11-08 14:12:53.4050 NZDT kernel.cc:1075] Use fast generic engine


i:22


[WARNING 23-11-08 14:12:53.8666 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:12:53.8666 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:12:53.8666 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:12:54.6760 NZDT kernel.cc:1243] Loading model from path /tmp/tmpqpgdh99d/model/ with prefix 7f915c1a74554d27
[INFO 23-11-08 14:12:54.6808 NZDT kernel.cc:1075] Use fast generic engine


i:23


[WARNING 23-11-08 14:12:55.0904 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:12:55.0905 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:12:55.0905 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:12:56.0651 NZDT kernel.cc:1243] Loading model from path /tmp/tmp8r12jwvv/model/ with prefix 2d83836e622e468b
[INFO 23-11-08 14:12:56.0720 NZDT kernel.cc:1075] Use fast generic engine


i:24


[WARNING 23-11-08 14:12:56.5313 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:12:56.5313 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:12:56.5313 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:12:57.2240 NZDT kernel.cc:1243] Loading model from path /tmp/tmpce0wfic3/model/ with prefix 980c6062a303450d
[INFO 23-11-08 14:12:57.2273 NZDT kernel.cc:1075] Use fast generic engine


i:25


[WARNING 23-11-08 14:12:57.5566 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:12:57.5567 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:12:57.5567 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:12:58.4733 NZDT kernel.cc:1243] Loading model from path /tmp/tmpafx_kuse/model/ with prefix 65c26201e3234850
[INFO 23-11-08 14:12:58.4815 NZDT kernel.cc:1075] Use fast generic engine


i:26


[WARNING 23-11-08 14:12:58.8610 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:12:58.8610 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:12:58.8610 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:12:59.7515 NZDT kernel.cc:1243] Loading model from path /tmp/tmpe4_rybrq/model/ with prefix a93f9154c97c46c6
[INFO 23-11-08 14:12:59.7579 NZDT kernel.cc:1075] Use fast generic engine


i:27


[WARNING 23-11-08 14:13:00.1121 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:00.1121 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:00.1121 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:13:00.8025 NZDT kernel.cc:1243] Loading model from path /tmp/tmpufscio_f/model/ with prefix d8a731b53beb487c
[INFO 23-11-08 14:13:00.8074 NZDT kernel.cc:1075] Use fast generic engine


i:28


[WARNING 23-11-08 14:13:01.4997 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:01.4998 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:01.4998 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:13:02.1579 NZDT kernel.cc:1243] Loading model from path /tmp/tmpbgrlsurn/model/ with prefix 70495f266bd742b6
[INFO 23-11-08 14:13:02.1614 NZDT kernel.cc:1075] Use fast generic engine


i:29


[WARNING 23-11-08 14:13:02.5515 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:02.5515 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:02.5515 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:13:03.6716 NZDT kernel.cc:1243] Loading model from path /tmp/tmp8j5nd81o/model/ with prefix d95d9c6efb5c4f19
[INFO 23-11-08 14:13:03.6817 NZDT abstract_model.cc:1311] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 23-11-08 14:13:03.6818 NZDT kernel.cc:1075] Use fast generic engine


i:30


[WARNING 23-11-08 14:13:04.1045 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:04.1046 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:04.1046 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:13:05.6913 NZDT kernel.cc:1243] Loading model from path /tmp/tmpyrha8kwj/model/ with prefix 0d06807e59c34d5a
[INFO 23-11-08 14:13:05.7082 NZDT kernel.cc:1075] Use fast generic engine


i:31


[WARNING 23-11-08 14:13:06.0488 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:06.0488 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:06.0488 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:13:06.8693 NZDT kernel.cc:1243] Loading model from path /tmp/tmp98rdjvl4/model/ with prefix 7ffa22ef45774839
[INFO 23-11-08 14:13:06.8750 NZDT kernel.cc:1075] Use fast generic engine


i:32


[WARNING 23-11-08 14:13:07.1819 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:07.1819 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:07.1819 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:13:07.7960 NZDT kernel.cc:1243] Loading model from path /tmp/tmpwbg0ta48/model/ with prefix 8ac9a33fe6fe4c18
[INFO 23-11-08 14:13:07.7993 NZDT kernel.cc:1075] Use fast generic engine


i:33


[WARNING 23-11-08 14:13:08.1198 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:08.1199 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:08.1199 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:13:09.1996 NZDT kernel.cc:1243] Loading model from path /tmp/tmp8cwtbgok/model/ with prefix f0fc5bed16ae4768
[INFO 23-11-08 14:13:09.2114 NZDT kernel.cc:1075] Use fast generic engine


i:34


[WARNING 23-11-08 14:13:09.6018 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:09.6018 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:09.6018 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:13:10.3700 NZDT kernel.cc:1243] Loading model from path /tmp/tmpvus9x57_/model/ with prefix 6240a931f4b84f6d
[INFO 23-11-08 14:13:10.3767 NZDT kernel.cc:1075] Use fast generic engine


i:35


[WARNING 23-11-08 14:13:10.7532 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:10.7532 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:10.7532 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:13:11.5463 NZDT kernel.cc:1243] Loading model from path /tmp/tmpvw646erq/model/ with prefix 3b7adc35db2b49a4
[INFO 23-11-08 14:13:11.5507 NZDT kernel.cc:1075] Use fast generic engine


i:36


[WARNING 23-11-08 14:13:11.9815 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:11.9815 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:11.9815 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:13:12.9963 NZDT kernel.cc:1243] Loading model from path /tmp/tmp_mito532/model/ with prefix 8ab6bb5a60a34362
[INFO 23-11-08 14:13:13.0046 NZDT kernel.cc:1075] Use fast generic engine


i:37


[WARNING 23-11-08 14:13:13.3907 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:13.3907 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:13.3907 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:13:14.2480 NZDT kernel.cc:1243] Loading model from path /tmp/tmp7j5mgl3y/model/ with prefix 3d139d3823cc40a7
[INFO 23-11-08 14:13:14.2530 NZDT abstract_model.cc:1311] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 23-11-08 14:13:14.2531 NZDT kernel.cc:1075] Use fast generic engine


i:38


[WARNING 23-11-08 14:13:14.6420 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:14.6421 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:14.6421 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:13:15.7068 NZDT kernel.cc:1243] Loading model from path /tmp/tmpq870d5wn/model/ with prefix 42805a28d56941a6
[INFO 23-11-08 14:13:15.7151 NZDT kernel.cc:1075] Use fast generic engine


i:39


[WARNING 23-11-08 14:13:16.1068 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:16.1068 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:16.1068 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:13:17.5643 NZDT kernel.cc:1243] Loading model from path /tmp/tmpnh9zs_ol/model/ with prefix cbcd1ffaac1849cc
[INFO 23-11-08 14:13:17.5732 NZDT kernel.cc:1075] Use fast generic engine


i:40


[WARNING 23-11-08 14:13:17.9805 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:17.9805 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:17.9805 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:13:18.6869 NZDT kernel.cc:1243] Loading model from path /tmp/tmpou6agv69/model/ with prefix 3c93edb62c7b4835
[INFO 23-11-08 14:13:18.6905 NZDT kernel.cc:1075] Use fast generic engine


i:41


[WARNING 23-11-08 14:13:19.1250 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:19.1250 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:19.1250 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:13:20.3557 NZDT kernel.cc:1243] Loading model from path /tmp/tmp2dufi2rh/model/ with prefix b8ee985070334185
[INFO 23-11-08 14:13:20.3646 NZDT kernel.cc:1075] Use fast generic engine


i:42


[WARNING 23-11-08 14:13:20.6977 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:20.6977 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:20.6977 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:13:21.6181 NZDT kernel.cc:1243] Loading model from path /tmp/tmpzzak5aza/model/ with prefix f3948ae92a0142db
[INFO 23-11-08 14:13:21.6240 NZDT kernel.cc:1075] Use fast generic engine


i:43


[WARNING 23-11-08 14:13:21.9557 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:21.9557 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:21.9557 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:13:23.2400 NZDT kernel.cc:1243] Loading model from path /tmp/tmpkjm9o58v/model/ with prefix fc09320fbbc041b0
[INFO 23-11-08 14:13:23.2516 NZDT kernel.cc:1075] Use fast generic engine


i:44


[WARNING 23-11-08 14:13:23.6414 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:23.6414 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:23.6414 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:13:24.5411 NZDT kernel.cc:1243] Loading model from path /tmp/tmpvk9fyj7i/model/ with prefix 29ffeeecd41f444f
[INFO 23-11-08 14:13:24.5482 NZDT abstract_model.cc:1311] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 23-11-08 14:13:24.5482 NZDT kernel.cc:1075] Use fast generic engine


i:45


[WARNING 23-11-08 14:13:24.8929 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:24.8929 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:24.8929 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:13:25.4580 NZDT kernel.cc:1243] Loading model from path /tmp/tmpxovp5oa2/model/ with prefix fad2e3ace3b045c3
[INFO 23-11-08 14:13:25.4603 NZDT kernel.cc:1075] Use fast generic engine


i:46


[WARNING 23-11-08 14:13:25.7703 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:25.7704 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:25.7704 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:13:27.0102 NZDT kernel.cc:1243] Loading model from path /tmp/tmpgpfjt5tn/model/ with prefix c9fd36cae16946c3
[INFO 23-11-08 14:13:27.0215 NZDT kernel.cc:1075] Use fast generic engine


i:47


[WARNING 23-11-08 14:13:27.4361 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:27.4362 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:27.4362 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:13:28.4140 NZDT kernel.cc:1243] Loading model from path /tmp/tmpislr7g58/model/ with prefix 57db045063854ddf
[INFO 23-11-08 14:13:28.4223 NZDT kernel.cc:1075] Use fast generic engine


i:48


[WARNING 23-11-08 14:13:28.7844 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:28.7844 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:28.7845 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:13:29.7614 NZDT kernel.cc:1243] Loading model from path /tmp/tmpv9_anl_g/model/ with prefix 8d94441824e64100
[INFO 23-11-08 14:13:29.7667 NZDT kernel.cc:1075] Use fast generic engine


i:49


[WARNING 23-11-08 14:13:30.1225 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:30.1225 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:30.1226 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:13:30.8533 NZDT kernel.cc:1243] Loading model from path /tmp/tmp6omi9u9i/model/ with prefix 8f874e1427ae406c
[INFO 23-11-08 14:13:30.8576 NZDT kernel.cc:1075] Use fast generic engine


i:50


[WARNING 23-11-08 14:13:31.2448 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:31.2448 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:31.2448 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:13:32.1533 NZDT kernel.cc:1243] Loading model from path /tmp/tmp6k4bnqpx/model/ with prefix 3a2c954079114ca1
[INFO 23-11-08 14:13:32.1627 NZDT kernel.cc:1075] Use fast generic engine


i:51


[WARNING 23-11-08 14:13:32.5266 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:32.5266 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:32.5266 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:13:33.6312 NZDT kernel.cc:1243] Loading model from path /tmp/tmpgm9prjcc/model/ with prefix e8487ed1f49f4d5b
[INFO 23-11-08 14:13:33.6446 NZDT kernel.cc:1075] Use fast generic engine


i:52


[WARNING 23-11-08 14:13:34.0353 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:34.0353 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:34.0353 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:13:34.7763 NZDT kernel.cc:1243] Loading model from path /tmp/tmpt476umzs/model/ with prefix 30e00e92a5714e6e
[INFO 23-11-08 14:13:34.7806 NZDT abstract_model.cc:1311] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 23-11-08 14:13:34.7807 NZDT kernel.cc:1075] Use fast generic engine


i:53


[WARNING 23-11-08 14:13:35.1344 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:35.1344 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:35.1344 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:13:35.8309 NZDT kernel.cc:1243] Loading model from path /tmp/tmps_uyepjj/model/ with prefix 03cdb80bf5e24e28
[INFO 23-11-08 14:13:35.8351 NZDT kernel.cc:1075] Use fast generic engine


i:54


[WARNING 23-11-08 14:13:36.2128 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:36.2128 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:36.2129 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:13:36.7806 NZDT kernel.cc:1243] Loading model from path /tmp/tmpe7ezfsgu/model/ with prefix fedaa7f8ba3845be
[INFO 23-11-08 14:13:36.7825 NZDT kernel.cc:1075] Use fast generic engine


i:55


[WARNING 23-11-08 14:13:37.1290 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:37.1290 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:37.1290 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:13:38.1035 NZDT kernel.cc:1243] Loading model from path /tmp/tmpdyu4esdi/model/ with prefix 1c780d1e5d1342dc
[INFO 23-11-08 14:13:38.1119 NZDT kernel.cc:1075] Use fast generic engine


i:56


[WARNING 23-11-08 14:13:38.4362 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:38.4362 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:38.4362 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:13:39.3433 NZDT kernel.cc:1243] Loading model from path /tmp/tmp0gg047qm/model/ with prefix 9313132a20194dbe
[INFO 23-11-08 14:13:39.3503 NZDT kernel.cc:1075] Use fast generic engine


i:57


[WARNING 23-11-08 14:13:39.6857 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:39.6857 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:39.6858 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:13:40.3223 NZDT kernel.cc:1243] Loading model from path /tmp/tmpgxoeg0aj/model/ with prefix 99dcf5a645424589
[INFO 23-11-08 14:13:40.3263 NZDT kernel.cc:1075] Use fast generic engine


i:58


[WARNING 23-11-08 14:13:40.6489 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:40.6489 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:40.6489 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:13:41.4190 NZDT kernel.cc:1243] Loading model from path /tmp/tmpyfxogfmn/model/ with prefix 72713d1a1177434a
[INFO 23-11-08 14:13:41.4263 NZDT kernel.cc:1075] Use fast generic engine


i:59


[WARNING 23-11-08 14:13:41.8614 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:41.8615 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:41.8615 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:13:42.6177 NZDT kernel.cc:1243] Loading model from path /tmp/tmp_zw3zj1n/model/ with prefix 73b3cab9350243e3
[INFO 23-11-08 14:13:42.6237 NZDT kernel.cc:1075] Use fast generic engine


i:60


[WARNING 23-11-08 14:13:42.9327 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:42.9327 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:42.9327 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:13:43.7440 NZDT kernel.cc:1243] Loading model from path /tmp/tmp5eiiad3e/model/ with prefix a22ccba2fd974463
[INFO 23-11-08 14:13:43.7498 NZDT kernel.cc:1075] Use fast generic engine


i:61


[WARNING 23-11-08 14:13:44.1286 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:44.1286 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:44.1286 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:13:44.8563 NZDT kernel.cc:1243] Loading model from path /tmp/tmpv5ym_d48/model/ with prefix 1549e3c942a04e90
[INFO 23-11-08 14:13:44.8600 NZDT abstract_model.cc:1311] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 23-11-08 14:13:44.8600 NZDT kernel.cc:1075] Use fast generic engine


i:62


[WARNING 23-11-08 14:13:45.2970 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:45.2970 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:45.2970 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:13:46.8214 NZDT kernel.cc:1243] Loading model from path /tmp/tmpygl1zvbh/model/ with prefix 1658c413ce4b4176
[INFO 23-11-08 14:13:46.8358 NZDT kernel.cc:1075] Use fast generic engine


i:63


[WARNING 23-11-08 14:13:47.1559 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:47.1559 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:47.1559 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:13:47.8648 NZDT kernel.cc:1243] Loading model from path /tmp/tmpy30iiy_t/model/ with prefix 7b5bd52caac24761
[INFO 23-11-08 14:13:47.8698 NZDT kernel.cc:1075] Use fast generic engine


i:64


[WARNING 23-11-08 14:13:48.2109 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:48.2109 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:48.2109 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:13:48.8469 NZDT kernel.cc:1243] Loading model from path /tmp/tmpxv20f58s/model/ with prefix 5e301936db854bcf
[INFO 23-11-08 14:13:48.8556 NZDT kernel.cc:1075] Use fast generic engine


i:65


[WARNING 23-11-08 14:13:49.2194 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:49.2194 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:49.2194 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:13:49.9341 NZDT kernel.cc:1243] Loading model from path /tmp/tmp39fd38tl/model/ with prefix ed01fe72f25047dc
[INFO 23-11-08 14:13:49.9385 NZDT kernel.cc:1075] Use fast generic engine


i:66


[WARNING 23-11-08 14:13:50.2550 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:50.2550 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:50.2550 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:13:50.8973 NZDT kernel.cc:1243] Loading model from path /tmp/tmpy5fepo68/model/ with prefix a873d7b9cec14edb
[INFO 23-11-08 14:13:50.9012 NZDT kernel.cc:1075] Use fast generic engine


i:67


[WARNING 23-11-08 14:13:51.2500 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:51.2500 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:51.2500 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:13:52.4350 NZDT kernel.cc:1243] Loading model from path /tmp/tmppamo1a6r/model/ with prefix 4f6756369aaf4409
[INFO 23-11-08 14:13:52.4456 NZDT kernel.cc:1075] Use fast generic engine


i:68


[WARNING 23-11-08 14:13:52.8929 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:52.8929 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:52.8929 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:13:53.8147 NZDT kernel.cc:1243] Loading model from path /tmp/tmpl0vyocex/model/ with prefix 6517532e30d94ada
[INFO 23-11-08 14:13:53.8215 NZDT kernel.cc:1075] Use fast generic engine


i:69


[WARNING 23-11-08 14:13:54.1691 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:54.1691 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:54.1691 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:13:54.8608 NZDT kernel.cc:1243] Loading model from path /tmp/tmph2tul0xn/model/ with prefix 2e1e94ffdb7a49f3
[INFO 23-11-08 14:13:54.8670 NZDT abstract_model.cc:1311] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 23-11-08 14:13:54.8670 NZDT kernel.cc:1075] Use fast generic engine


i:70


[WARNING 23-11-08 14:13:55.2106 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:55.2106 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:55.2106 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:13:55.9567 NZDT kernel.cc:1243] Loading model from path /tmp/tmpouattdts/model/ with prefix ebe9e18396ee4385
[INFO 23-11-08 14:13:55.9621 NZDT kernel.cc:1075] Use fast generic engine


i:71


[WARNING 23-11-08 14:13:56.6188 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:56.6188 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:56.6188 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:13:57.3528 NZDT kernel.cc:1243] Loading model from path /tmp/tmpnljm7ec9/model/ with prefix 573765d38f5a4fcb
[INFO 23-11-08 14:13:57.3579 NZDT kernel.cc:1075] Use fast generic engine


i:72


[WARNING 23-11-08 14:13:57.6701 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:57.6701 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:57.6701 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:13:58.6952 NZDT kernel.cc:1243] Loading model from path /tmp/tmpt6rlwpq5/model/ with prefix 8257b07988f44f51
[INFO 23-11-08 14:13:58.7079 NZDT kernel.cc:1075] Use fast generic engine


i:73


[WARNING 23-11-08 14:13:59.0934 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:59.0934 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:13:59.0934 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:13:59.7868 NZDT kernel.cc:1243] Loading model from path /tmp/tmp26jzguz9/model/ with prefix c48ff8b552c540a8
[INFO 23-11-08 14:13:59.7913 NZDT kernel.cc:1075] Use fast generic engine


i:74


[WARNING 23-11-08 14:14:00.1356 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:14:00.1356 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:14:00.1356 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:14:00.9896 NZDT kernel.cc:1243] Loading model from path /tmp/tmpmk0zmmhj/model/ with prefix 7398fada96424588
[INFO 23-11-08 14:14:00.9976 NZDT kernel.cc:1075] Use fast generic engine


i:75


[WARNING 23-11-08 14:14:01.3741 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:14:01.3741 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:14:01.3741 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:14:02.1533 NZDT kernel.cc:1243] Loading model from path /tmp/tmprosj0i1j/model/ with prefix cfcf6d75f348443e
[INFO 23-11-08 14:14:02.1579 NZDT kernel.cc:1075] Use fast generic engine


i:76


[WARNING 23-11-08 14:14:02.4985 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:14:02.4986 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:14:02.4986 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:14:03.0913 NZDT kernel.cc:1243] Loading model from path /tmp/tmpspsy8jvv/model/ with prefix b96c34f4401e4d81
[INFO 23-11-08 14:14:03.0935 NZDT kernel.cc:1075] Use fast generic engine


i:77


[WARNING 23-11-08 14:14:03.4024 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:14:03.4025 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:14:03.4025 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:14:04.0885 NZDT kernel.cc:1243] Loading model from path /tmp/tmpeat61auy/model/ with prefix 428c017c4c5d473c
[INFO 23-11-08 14:14:04.0960 NZDT kernel.cc:1075] Use fast generic engine


i:78


[WARNING 23-11-08 14:14:04.4659 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:14:04.4659 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:14:04.4659 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:14:05.1940 NZDT kernel.cc:1243] Loading model from path /tmp/tmp_cp06giy/model/ with prefix b24efc1ac8784f81
[INFO 23-11-08 14:14:05.1980 NZDT abstract_model.cc:1311] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 23-11-08 14:14:05.1980 NZDT kernel.cc:1075] Use fast generic engine


i:79


[WARNING 23-11-08 14:14:05.5102 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:14:05.5102 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:14:05.5102 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:14:06.1846 NZDT kernel.cc:1243] Loading model from path /tmp/tmpsrska55u/model/ with prefix ca674d824ac64062
[INFO 23-11-08 14:14:06.1884 NZDT kernel.cc:1075] Use fast generic engine


i:80


[WARNING 23-11-08 14:14:06.5442 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:14:06.5442 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:14:06.5442 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:14:07.3168 NZDT kernel.cc:1243] Loading model from path /tmp/tmp6w5qw96v/model/ with prefix 51f8dd9b9a9c4ea0
[INFO 23-11-08 14:14:07.3220 NZDT kernel.cc:1075] Use fast generic engine


i:81


[WARNING 23-11-08 14:14:07.6089 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:14:07.6089 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:14:07.6089 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:14:08.4294 NZDT kernel.cc:1243] Loading model from path /tmp/tmpzkw1gm_d/model/ with prefix 354d8169f57343ff
[INFO 23-11-08 14:14:08.4354 NZDT kernel.cc:1075] Use fast generic engine


i:82


[WARNING 23-11-08 14:14:08.7282 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:14:08.7282 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:14:08.7282 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:14:09.5403 NZDT kernel.cc:1243] Loading model from path /tmp/tmp3vrsp23v/model/ with prefix 91891a7096da49ff
[INFO 23-11-08 14:14:09.5507 NZDT kernel.cc:1075] Use fast generic engine


i:83


[WARNING 23-11-08 14:14:09.8819 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:14:09.8819 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:14:09.8819 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:14:10.6141 NZDT kernel.cc:1243] Loading model from path /tmp/tmps_u3mr5k/model/ with prefix d5b33c231e7845a2
[INFO 23-11-08 14:14:10.6193 NZDT kernel.cc:1075] Use fast generic engine


i:84


[WARNING 23-11-08 14:14:10.9630 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:14:10.9630 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:14:10.9630 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:14:12.0007 NZDT kernel.cc:1243] Loading model from path /tmp/tmped59j9ww/model/ with prefix c4cd04e33bd94fa0
[INFO 23-11-08 14:14:12.0105 NZDT kernel.cc:1075] Use fast generic engine


i:85


[WARNING 23-11-08 14:14:12.3617 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:14:12.3617 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:14:12.3617 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:14:12.9632 NZDT kernel.cc:1243] Loading model from path /tmp/tmpfla1_ky4/model/ with prefix f832cafed3e4421d
[INFO 23-11-08 14:14:12.9672 NZDT kernel.cc:1075] Use fast generic engine


i:86


[WARNING 23-11-08 14:14:13.2822 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:14:13.2823 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:14:13.2823 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:14:14.3468 NZDT kernel.cc:1243] Loading model from path /tmp/tmpa1mwsyt2/model/ with prefix acb781931634440c
[INFO 23-11-08 14:14:14.3556 NZDT kernel.cc:1075] Use fast generic engine


i:87


[WARNING 23-11-08 14:14:14.6978 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:14:14.6978 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:14:14.6978 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:14:15.8405 NZDT kernel.cc:1243] Loading model from path /tmp/tmph2567bt_/model/ with prefix 1872661360d549cb
[INFO 23-11-08 14:14:15.8504 NZDT abstract_model.cc:1311] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 23-11-08 14:14:15.8504 NZDT kernel.cc:1075] Use fast generic engine


i:88


[WARNING 23-11-08 14:14:16.2122 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:14:16.2123 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:14:16.2123 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:14:17.0938 NZDT kernel.cc:1243] Loading model from path /tmp/tmpld1vfr5q/model/ with prefix a861394d5de64ad2
[INFO 23-11-08 14:14:17.1007 NZDT kernel.cc:1075] Use fast generic engine


i:89


[WARNING 23-11-08 14:14:17.4982 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:14:17.4982 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:14:17.4982 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:14:18.1668 NZDT kernel.cc:1243] Loading model from path /tmp/tmpvm2xe8lr/model/ with prefix 9d3223d41dd04746
[INFO 23-11-08 14:14:18.1700 NZDT kernel.cc:1075] Use fast generic engine


i:90


[WARNING 23-11-08 14:14:18.5916 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:14:18.5916 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:14:18.5916 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:14:19.5302 NZDT kernel.cc:1243] Loading model from path /tmp/tmpdgtbo15b/model/ with prefix 47bda43443e847f8
[INFO 23-11-08 14:14:19.5360 NZDT kernel.cc:1075] Use fast generic engine


i:91


[WARNING 23-11-08 14:14:19.8911 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:14:19.8911 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:14:19.8911 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:14:20.6329 NZDT kernel.cc:1243] Loading model from path /tmp/tmpbfd15ijj/model/ with prefix 321eacb80c254971
[INFO 23-11-08 14:14:20.6367 NZDT kernel.cc:1075] Use fast generic engine


i:92


[WARNING 23-11-08 14:14:21.1984 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:14:21.1985 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:14:21.1985 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:14:23.0040 NZDT kernel.cc:1243] Loading model from path /tmp/tmpt4kaigkb/model/ with prefix ae222cffa1c74d00
[INFO 23-11-08 14:14:23.0423 NZDT kernel.cc:1075] Use fast generic engine


i:93


[WARNING 23-11-08 14:14:23.7314 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:14:23.7315 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:14:23.7315 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:14:24.7050 NZDT kernel.cc:1243] Loading model from path /tmp/tmp5i2j7527/model/ with prefix c3664fa7787b4f70
[INFO 23-11-08 14:14:24.7107 NZDT kernel.cc:1075] Use fast generic engine


i:94


[WARNING 23-11-08 14:14:25.1292 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:14:25.1292 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:14:25.1292 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:14:25.9488 NZDT kernel.cc:1243] Loading model from path /tmp/tmpdwjqtiqh/model/ with prefix d1cca2fe0e1944e8
[INFO 23-11-08 14:14:25.9523 NZDT abstract_model.cc:1311] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 23-11-08 14:14:25.9523 NZDT kernel.cc:1075] Use fast generic engine


i:95


[WARNING 23-11-08 14:14:26.2578 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:14:26.2578 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:14:26.2578 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:14:27.2842 NZDT kernel.cc:1243] Loading model from path /tmp/tmpbuwtt2cd/model/ with prefix d71d1324f2094e13
[INFO 23-11-08 14:14:27.2908 NZDT kernel.cc:1075] Use fast generic engine


i:96


[WARNING 23-11-08 14:14:27.6237 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:14:27.6238 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:14:27.6238 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:14:28.3598 NZDT kernel.cc:1243] Loading model from path /tmp/tmptwwy7iap/model/ with prefix d745eca8644c460d
[INFO 23-11-08 14:14:28.3665 NZDT kernel.cc:1075] Use fast generic engine


i:97


[WARNING 23-11-08 14:14:28.6948 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:14:28.6948 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:14:28.6948 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:14:29.3341 NZDT kernel.cc:1243] Loading model from path /tmp/tmpatrcmq23/model/ with prefix 0146fd976f9e4cf1
[INFO 23-11-08 14:14:29.3376 NZDT kernel.cc:1075] Use fast generic engine


i:98


[WARNING 23-11-08 14:14:29.6432 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:14:29.6432 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:14:29.6432 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:14:30.3045 NZDT kernel.cc:1243] Loading model from path /tmp/tmpe1_hdf2s/model/ with prefix 26524fd2921c4ba8
[INFO 23-11-08 14:14:30.3090 NZDT kernel.cc:1075] Use fast generic engine


i:99


[WARNING 23-11-08 14:14:30.6325 NZDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:14:30.6325 NZDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-11-08 14:14:30.6325 NZDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-11-08 14:14:31.5680 NZDT kernel.cc:1243] Loading model from path /tmp/tmpvhidmg9j/model/ with prefix 1f50e646a40842c8
[INFO 23-11-08 14:14:31.5766 NZDT kernel.cc:1075] Use fast generic engine


Submission exported to Dataset/submission.csv


# What is next

If you want to learn more about TensorFlow Decision Forests and its advanced features, you can follow the official documentation [here](https://www.tensorflow.org/decision_forests) 